editors/linux-wps-office/files/patch-usr_bin_et

```diff
@@ -0,0 +1,11 @@
+--- usr/bin/et.orig	2021-06-02 03:31:05 UTC
++++ usr/bin/et
+@@ -7,7 +7,7 @@ gBinPath=$(dirname "$0")
+ if [ -d "${gBinPath}/office6" ]; then
+ 	gInstallPath=${gBinPath}
+ else
+-	gInstallPath=/opt/kingsoft/wps-office
++	gInstallPath=/compat/linux/opt/kingsoft/wps-office
```

Comment:
`/compat/linux` should be `%%LINUXBASE%%` and replaced with `${LINUXBASE}` in `post-patch:` target.

Another way is just using replace to `s,gInstallPath=/opt/,gInstallPath=${LINUXBASE}/opt/,` for those files in `post-patch:` target. This might be a cleaner way.

But I still prefer if upstream can modify the code to support different install prefix.

Reply:
Ah, will do.

> But I still prefer if upstream can modify the code to support different install prefix.

Commercial software from mainland, not really an upstream ;)